backend/src/main/java/com/soen341/instagram/exception/like/MultipleLikeException.java

```diff
@@ -8,7 +8,7 @@
 {
 	public MultipleLikeException()
 	{
-		super("The content can only be liked once by the user");
+		super("There can't be more than one like per user.");
```

Comment:
I think the error message should be as if you were talking to the user. Because the frontend will receive the message and (I think) we will display the error to the user. So they should see something like `You have already liked this.` rather than `There cannot be more than one like per user.`